test/test_with_xpress.jl

```diff
@@ -994,6 +994,79 @@ end
 
 end
 
+@testset "GHP" begin
+    """
+
+    This tests multiple unique aspects of GHP:
+    1. REopt.jl calls GhpGhx.jl n times from within scenario.jl and creates a list of n GHP structs
+    2. GHP with heating and cooling "..efficiency_thermal_factors" reduces the net thermal load
+    3. GHP serves only the SpaceHeatingLoad by default unless it is allowed to serve DHW
+    4. GHP serves all the Cooling load
+    5. Input of a custom COP map for GHP and check the GHP performance to make sure it's using it correctly
+
+    TODO once these separate features are added to REopt.jl:
+    6. Alternative schedule-based FlatLoad profiles are used
+    7. "Addressable load fraction" reduces the total heating load relative to the fuel input
+    
+    """
+    # Load base inputs
+    input_data = JSON.parsefile("scenarios/test_ghp_inputs.json")
+    
+    # Modify ["GHP"]["ghpghx_inputs"] for running GhpGhx.jl
+    # Heat pump performance maps
+    cop_map_df = CSV.read("scenarios/ghp_cop_map_custom.csv", DataFrame)
+    # Generate a "records" style dictionary from the 
+    cop_map_list = []
+    for i in eachrow(cop_map_df)
+        dict_record = Dict(names(i)[k]=>i[names(i)[k]] for k in 1:length(names(i)))
+        push!(cop_map_list, dict_record)
+    end
+    input_data["GHP"]["ghpghx_inputs"][1]["cop_map_eft_heating_cooling"] = cop_map_list
```

Comment:
Since we don't use CSV or DataFrames anywhere else, and `scenarios/ghp_cop_map_custom.csv` is two rows X three columns, can this just be put in JSON format instead to match the other input files?  Or maybe combine it with the other GHP inputs instead?  (If we do so, the deps and includes for CSV and DataFrame should be removed from test/project.toml and tes/runtests.jl, respectively.

Reply:
I kept the data file as .csv, and I'm now using what we currently use for Wind to parse the data file using the standard library DelimitedFiles package.

Reply:
I like the new solution - thanks for this!